src/cmake/dependencies.cmake

```diff
@@ -130,78 +173,176 @@ endif()
 
 # Magnum. Use a system package, if preferred.
 if(NOT USE_SYSTEM_MAGNUM)
-  set(BUILD_PLUGINS_STATIC ON CACHE BOOL "BUILD_PLUGINS_STATIC" FORCE)
-  set(BUILD_STATIC ON CACHE BOOL "BUILD_STATIC" FORCE)
-  set(BUILD_STATIC_PIC ON CACHE BOOL "BUILD_STATIC_PIC" FORCE)
+  set(BUILD_PLUGINS_STATIC
+      ON
+      CACHE BOOL "BUILD_PLUGINS_STATIC" FORCE
+  )
+  set(BUILD_STATIC
+      ON
+      CACHE BOOL "BUILD_STATIC" FORCE
+  )
+  set(BUILD_STATIC_PIC
+      ON
+      CACHE BOOL "BUILD_STATIC_PIC" FORCE
+  )
 
   # These are enabled by default but we don't need them right now -- disabling
   # for slightly faster builds. If you need any of these, simply delete a line.
-  set(WITH_TEXT OFF CACHE BOOL "" FORCE)
-  set(WITH_TEXTURETOOLS OFF CACHE BOOL "" FORCE)
+  set(WITH_TEXT
+      OFF
+      CACHE BOOL "" FORCE
+  )
+  set(WITH_TEXTURETOOLS
+      OFF
+      CACHE BOOL "" FORCE
+  )
 
   # These are not enabled by default but we need them
-  set(WITH_ANYSCENEIMPORTER ON CACHE BOOL "WITH_ANYSCENEIMPORTER" FORCE)
-  if (BUILD_ASSIMP_SUPPORT)
-    set(WITH_ASSIMPIMPORTER ON CACHE BOOL "WITH_ASSIMPIMPORTER" FORCE)
+  set(WITH_ANYSCENEIMPORTER
+      ON
+      CACHE BOOL "WITH_ANYSCENEIMPORTER" FORCE
+  )
+  if(BUILD_ASSIMP_SUPPORT)
+    set(WITH_ASSIMPIMPORTER
+        ON
+        CACHE BOOL "WITH_ASSIMPIMPORTER" FORCE
+    )
   endif()
-  set(WITH_TINYGLTFIMPORTER ON CACHE BOOL "WITH_TINYGLTFIMPORTER" FORCE)
-  set(WITH_ANYIMAGEIMPORTER ON CACHE BOOL "WITH_ANYIMAGEIMPORTER" FORCE)
-  set(WITH_ANYIMAGECONVERTER ON CACHE BOOL "WITH_ANYIMAGECONVERTER" FORCE)
-  set(WITH_PRIMITIVEIMPORTER ON CACHE BOOL "" FORCE)
-  set(WITH_STANFORDIMPORTER ON CACHE BOOL "" FORCE)
-  set(WITH_STBIMAGEIMPORTER ON CACHE BOOL "WITH_STBIMAGEIMPORTER" FORCE)
-  set(WITH_STBIMAGECONVERTER ON CACHE BOOL "WITH_STBIMAGECONVERTER" FORCE)
-  set(WITH_EMSCRIPTENAPPLICATION OFF CACHE BOOL "WITH_EMSCRIPTENAPPLICATION" FORCE)
-  set(WITH_GLFWAPPLICATION OFF CACHE BOOL "WITH_GLFWAPPLICATION" FORCE)
-  set(WITH_EIGEN ON CACHE BOOL "WITH_EIGEN" FORCE) # Eigen integration
-  set(WITH_IMGUI ON CACHE BOOL "WITH_IMGUI" FORCE) # ImGui integration
+  set(WITH_TINYGLTFIMPORTER
```

Comment:
@erikwijmans It's expressions like this that make prefer the default limit of 6 args per line instead of 3.